dev/cookbooks/provisioning/files/default/ldap-data/user-child.ldif

```diff
@@ -0,0 +1,12 @@
+dn: cn=Julia Child,ou=chefs,dc=chef-server,dc=dev
```

Comment:
request for `user-user1` ldif file too, with password `password`.  Keeps it simple and easy to remember when we want to test something quickly.